CHANGELOG.md

```diff
@@ -6,6 +6,10 @@ and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.
 
 ## [Unreleased]
 
+## [v0.2.0](https://github.com/cyberark/terraform-provider-conjur/releases/tag/v0.2.0) - 2018-08-29
```

Comment:
In other changelogs, the version links have pointed to a diff with the previous version. Should we do that here, too?

Reply:
i'm not sure. whatever we do we should adopt it team-wide, because i know several other projects do it like this. 

https://keepachangelog.com/en/1.0.0/ provides no guidance here, so I guess we have to pick. But I'd rather not leave this PR dangling while we work on consensus.

Reply:
My vote would be for the other style. GH already provides a view that gives links to tagged versions, like this one for conjur-api-go:  https://github.com/cyberark/conjur-api-go/tags . AFAIK, it doesn't have a view with links to diffs.

I guess this is fine for now.